Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.World.cpp

```diff
@@ -106,6 +106,7 @@ int CLuaFunctionDefs::GetRoofPosition(lua_State* luaVM)
         }
     }
     else
+        return luaL_error(luaVM, argStream.GetFullErrorMessage());
         m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
 
     // Return false
```

Comment:
I suggest changing this to:
```cpp
    if (argStream.HasErrors())
		return luaL_error(luaVM, argStream.GetFullErrorMessage());

    // Get the ground position and return it
    bool bOutResult;
    float fRoof = g_pGame->GetWorld()->FindRoofZFor3DCoord(&vecStart, &bOutResult);
    if (bOutResult)
    {
        lua_pushnumber(luaVM, fRoof);
        return 1;
    }

    lua_pushboolean(luaVM, false);
    return 1;
```

Reply:
Hmm I see, will revise. Also, I realize I totally forgot to remove the warning just after, whoops.